pennylane/ops/arithmetic/sum.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This file contains the implementation of the Sum class which contains logic for
+computing the sum of operations.
+"""
+import warnings
+import numpy as np
+import pennylane as qml
+from pennylane import numpy as qnp
+
+
+def sum(*summands):
+    """Top level function computing the sum of the
```

Comment:
```suggestion
    """Top-level function computing the sum of the
```

Reply:
As far as I am concerned, this is the main user-facing function (called by ``Operator.__add__``). 

I think a descriptive docstring would be great, for example saying that this creates an instance of the symbolic `Sum` class and specifying what ``summands`` is.

Reply:
Summands is an english word, no? I translated it from German "Summanden" but not sure how common it is? :)

Reply:
:+1: this should fit our existing docstring standards :)

Reply:
The term "summands" is uncommon enough that we should probably avoid it. I think we can use "terms" instead.

Reply:
Oh really? I don't think as `summands` as uncommon at all, it was used a lot in our university courses in Australia